### DISCUSSION TITLE
What is classed as an "automatic transaction" when it comes to notifications?

### DISCUSSION THREAD

**Discussion by digitlength** [firefly-iii/firefly-iii#8569] [Firefly III] [answered]
I have setup email notifications, which works when pressing the "Send message" button in the Administration settings page - I receive an email in my inbox.

Since then, however, I have not received any notifications. I am successfully importing transactions via the importer through GoCardless, triggered periodically via cron. I have had a number of new transactions added since setting up the email stuff. But I'm not getting any emails. 

I had assumed that "Alert when a transaction is created automatically" would send an email notification when a new transaction is added through the importer - is that assumption not correct?

**Comment by JC5** [comment, marked as answer]
The data importer has its own email configuration, you can find the details in the [.env.example](https://github.com/firefly-iii/data-importer/blob/main/.env.example#L193) file of the data importer.

The automated transactions you refer to are recurring transactions.
